lib/execution_engine2/utils/SDKMethodRunner.py

```diff
@@ -467,3 +467,20 @@ def update_job_status(self, job_id, status):
             job.save()
 
         return str(job.id)
+
+    def get_job_status(self, job_id):
+        returnVal = dict()
+
+        if not job_id:
+            raise ValueError("Please provide valid job_id")
+
+        with self.get_mongo_util().me_collection(self.config["mongo-jobs-collection"]):
+
+            try:
+                job = Job.objects(id=job_id)[0]
```

Comment:
I think its better to use  `self.get_mongo_util().me_collection.get_job` to keep database stuff in that utils file

Reply:
👍